ObelusFamily/Anythink-Market-IOJ5i
# Intial setup

# Description

Updated the project files to allow running on the local environment.
I tackled three types of errors:

1. running 'npm install' failed because react, react-dom and react-redux versions were outdated
2. backend failed to start because MONGODB_URI environment variable have not been defined.
3. frontend failed to start because the default port was not set to 3001

Fixes:

1. Modified react, react-dom and react-redux version in frontend package.json
2. Added .env file in backend directory with default MONGO_URL
3. Added .env.local file in frontend directory with default PORT set to 3001

# How Has This Been Tested?

Tested using the following steps:
1. Cloned a new repository
2. Installed all dependencies
3. Run start.sh under root folder
4. Make sure frontend client is accessible under http://localhost:3001 and new user can be created
